client/my-sites/customer-home/main.jsx

```diff
@@ -93,6 +94,7 @@ const Home = ( {
 						displayChecklist={ displayChecklist }
 					/>
 					<Upsells cards={ layout.upsells } />
+					{ ! displayChecklist && <ConnectAccountsTask /> }
```

Comment:
Let's make sure we drop this before merging this PR, so we don't render the card until the tasks container is ready and the Home layout API includes it in its response.